Rakefile

```diff
@@ -1,68 +1,6 @@
-require_relative 'config/application'
-require 'rspec/core/rake_task'
+# Add your own tasks in files placed in lib/tasks ending in .rake,
+# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-begin
-  Bundler.setup(:default, :development)
-rescue Bundler::BundlerError => e
-  $stderr.puts e.message
-  $stderr.puts "Run `bundle install` to install missing gems"
-  exit e.status_code
-end
+require_relative 'config/application'
 
 Rails.application.load_tasks
-
-Dir.glob('lib/tasks/*.rake').each { |r| import r }
-
-desc "Run specs"
-RSpec::Core::RakeTask.new(:spec)
-
-task :default => :spec
-
-if ['test', 'development'].include? ENV['RAILS_ENV']
-  require 'jettywrapper'
-  Jettywrapper.hydra_jetty_version = 'v7.3.0' # this keeps us on fedora 3, hydra-jetty v8.x moves to fedora 4.
-
-  def load_order_files(fedora_files)
-    data_path = File.expand_path('../fedora_conf/data/', __FILE__)
-    fedora_files.delete_if { |f| f.strip.empty? }
-    fedora_files.map { |f| File.join(data_path, f.strip) }
-  end
-
-  namespace :repo do
```

Comment:
so this repo:load task is useless to us?  and should be removed from README, if it's there?

Reply:
i was going to ask @blalbrit and @peetucket tomorrow (8/29) during standup

Reply:
During standup (8/29) it was decided that we can remove this rake task!